ui/src/app/utils/index.tsx

```diff
@@ -353,3 +353,25 @@ export const toggleIncludes = fp.curry((value, arr) => {
 export function sliceByHalfLength(obj) {
   return Math.ceil(obj.length / 2);
 }
+
+export function debouncer(action, sensitivityMs) {
```

Comment:
This could also use some documentation, the concept of "debouncing" is fairly common but I don't think the details can be assumed